.github/workflows/validate_release.yml

```
@@ -0,0 +1,62 @@
+name: Validate release
+
+on:
+  workflow_call:
+    inputs:
+      path:
+        description: 'Go mod directory (e.g. "plugins/source/aws", "plugins/destination/csv", "cli")'
```

```suggestion
        description: 'Go mod directory (e.g. "plugins/source/aws", "plugins/destination/file", "cli")'
```

The CSV plugin is deprecated so let's not use it in an example